### PR TITLE
data:Chetax

### PR DESCRIPTION
**Pull request by Chetax**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5055"></a>

### CODE REVIEW

File: Chetax.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Chetan Padhen",
+  "diplayStatsPublic": true,
+  "type": "personal",
+  "bio": " I am Btech CSE Student | Web2 developer,newbie To Web3 And Devops |Like  Contribution To **Open Source Project** ",
+  "avatar":"https://github.com/Chetax.png",
```
Comment (priyankarpal) [inline Chetax.json:6]: please remove the `avatar` code. Now it's no longer needed. 
```suggestion
```

File: Chetax.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Chetan Padhen",
+  "diplayStatsPublic": true,
```
Comment (priyankarpal) [inline Chetax.json:3]: Also, remove `  "diplayStatsPublic": true,`
```suggestion
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
Please move `Chetax.json` to `data/Chetax.json`

Screenshot from 2023-02-25 15-40-40

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by swagataroy30** [comment]
Hey but the `json` file is not in the `data` folder.

**Comment by priyankarpal** [comment]
hi @SaraJaoude please check json file is not in the data folder

**Comment by eddiejaoude** [comment]
Good spot 👍  we will fix

**Comment by eddiejaoude** [comment]
Fixed here https://github.com/EddieHubCommunity/LinkFree/pull/5105

**Pull request merged**